language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/workspace/TestWorkspaceManager.java

```diff
@@ -401,6 +402,24 @@ public void testWSEventsDeleteModulesDir() throws WorkspaceDocumentException, IO
         }
     }
 
+    @Test
+    public void testWSLoadProject() throws WorkspaceDocumentException, ProjectException {
+        Path filePath1 = RESOURCE_DIRECTORY.resolve("myproject").resolve("main.bal").toAbsolutePath();
+        Path filePath2 = RESOURCE_DIRECTORY.resolve("myproject2").resolve("main.bal").toAbsolutePath();
+
```

Comment:
Shall we add an assertion to check if the project is not there in the workspace manager?